Payments.Web/Startup.cs

```diff
@@ -32,9 +33,12 @@ public void ConfigureServices(IServiceCollection services)
 
             var connection = @"Server=.\sqlexpress;Database=Payments;Trusted_Connection=True;";
             services
-                .AddEntityFramework()
-                .AddEntityFrameworkSqlServer()
+                //.AddEntityFramework()
+                //.AddEntityFrameworkSqlServer()
                 .AddDbContext<PaymentsContext>(options => options.UseSqlServer(connection));
+
+            // automapper services
+            services.AddAutoMapper();
```

Comment:
@srkirkland It does an assembly scan per:
https://github.com/AutoMapper/AutoMapper.Extensions.Microsoft.DependencyInjection